### ISSUE TITLE
Provide option to generate sample objects

### ISSUE THREAD

**Issue by mjordan** [mjordan/islandora_workbench#58]
https://github.com/mjordan/islandora_scg fills this need in Islandora 7. Might be useful to provide an option to have Workbench ingest some random or semi-random nodes.

**Comment by mjordan** [comment]
Devel module does this already, but I haven't tried it.

**Comment by mjordan** [comment]
Some work in the abandoned https://github.com/mjordan/drupal_field_limit_tester/tree/islandora branch. Could repurpose for this.

**Comment by mjordan** [comment]
c5f13ba828ab26c2f055f6783d84e4f8bed9237b added a 'generate_image_files.py' bootstrap script. Requires ImageMagic's `convert` is present. Here's a config that will use that script to generate and ingest some sample images:

```yaml
task: create_from_files
host: "http://localhost:8000"
username: admin
password: islandora
input_dir: /tmp/output
media_use_tid: 17
output_csv: /tmp/issue-58-output.csv
content_type: islandora_object
drupal_filesystem: "fedora://"
media_types:
 - file: ['tif', 'tiff', 'jp2', 'zip', 'tar']
 - document: ['pdf', 'doc', 'docx', 'ppt', 'pptx']
 - image: ['png', 'gif', 'jpg', 'jpeg']
 - audio: ['mp3', 'wav', 'aac']
 - video: ['mp4']
 - extracted_text: ['txt']
# field_model: 23
models:
 - 23: ['zip', 'tar', '']
 - 27: ['pdf', 'doc', 'docx', 'ppt', 'pptx']
 - 25: ['tif', 'tiff', 'jp2', 'png', 'gif', 'jpg', 'jpeg']
 - 22: ['mp3', 'wav', 'aac']
 - 26: ['mp4']
bootstrap:
 - "/home/mark/Documents/hacking/workbench/generate_image_files.py"
```

The output directory configured in the script needs to match the value of `input_dir` in the config.

**Comment by mjordan** [comment]
Documented, and closed, with d319252f71dc97ebac050d161100e3aed28d65b8.
